src/setup.c

```diff
@@ -169,8 +169,6 @@ static void setup_speaker(void)
 
 	timer_disable_oc_output(TIM11, TIM_OC1);
 	timer_set_oc_mode(TIM11, TIM_OC1, TIM_OCM_PWM1);
-
-	timer_enable_break_main_output(TIM11);
```

Comment:
Remove also from the function comments

Reply:
Good catch! Updated in e1b38b9828d3ba3fcec37b1978aebfb2feb054a7

Reply:
ok